src/webapp/public/locales/en/translation.json

```diff
@@ -138,13 +162,13 @@
   "player": {
     "controls": {
       "shuffle": {
-        "activate": "Activate shuffle",
-        "deactivate": "Deactivate shuffle"
+        "enable": "Enable shuffle",
+        "disable": "Disable shuffle"
       },
-      "skip": "Skip previous track",
+      "prev_song": "Skip to previous track",
       "play": "Play",
       "pause": "Pause",
-      "next": "Skip next track",
+      "next_song": "Skip to next track",
       "repeat": {
         "activate": "Activate repeat",
```

Comment:
also rename to `enable / disable` like `shuffle` ?

Reply:
Should be. good catch